cmd/aries-js-worker/src/worker-impl-rest.js

```diff
@@ -67,4 +74,18 @@ function newResponse(id, payload, errMsg, topic) {
         errMsg: errMsg,
         topic: topic
     };
-}
\ No newline at end of file
+}
+
+const wsnotifier = class {
+    constructor(url, postMsg) {
+        this.socket = new WebSocket(url);
+        this.socket.addEventListener('message', function (event) {
+            // TODO REST agents are not currently revealing topic information on incoming messages,
+            //  Once REST supports this feature, topic value will be dynamic.
+            postMsg(newResponse("random-id",  event.data, "", "all"));
+        });
+    }
+    stop(){
+        this.socket.close(1011, "stopped notifier in aries-js-worker")
```

Comment:
1011?

can we replace magic number with a named const?

Reply:
Also, you should patch REST agent to not throw a log on 1011 status code.

Reply:
https://github.com/hyperledger/aries-framework-go/blob/master/pkg/controller/webnotifier/websocket.go#L94

Reply:
Never mind. Isn't 1011 an error code (https://github.com/nhooyr/websocket/blob/94f9b71576625ef4dbe5950253cbfa8b539b88b5/close.go#L42)

Shouldn't you use a non-error status?

Reply:
using normal closure [code](https://tools.ietf.org/html/rfc6455#section-7.4.1)